packages/server/lib/plugins/child/run_plugins.js

```diff
@@ -180,7 +180,7 @@ module.exports = (ipc, pluginsFile, projectRoot) => {
   if (!tsRegistered) {
     try {
       const tsPath = resolve.sync('typescript', {
-        basedir: this.projectRoot,
```

Comment:
`this.` is meaningless because it's an arrow function. It made projects load typescript from the `cypress` package, not their project.

Reply:
yeah, between and `project.js` and `resolve.js` the duplication is hard to manage. Also, the tsnode options are better covered in https://github.com/cypress-io/cypress/pull/7166